docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you need to create a tunnel to the coordinator pod and your computer. You can do
+this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       POD_NAME=$(kubectl get pods -l "app=trino,release=my-trino-cluster,component=coordinator" -o name)
+
+#. Create the tunnel from the coordinator pod to the client.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+   Now you can connect to the Trino coordinator at ``http://localhost:8080``.
+
+#. To connect to Trino, you can use the
+   :doc:`commond-line interface </client/cli>`, a
+   :doc:`jdbc client </client/jdbc>`, or any of the
+   :doc:`other clients </client>`. For this example,
+   :ref:`install the command-line interface <cli-installation>`, and connect to
+   Trino in a new console session.
+
+   .. code-block:: text
+
+       trino --server http://localhost:8080
+
+#. Using the sample data in the ``tpch`` catalog, type and execute a query on
+   the ``nation`` table using the ``tiny`` schema. Try other
+   commands to explore this data set:
+
+   .. code-block:: text
+
+       trino> select count(*) from tpch.tiny.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command in the
+   CLI.
+
+#. Kill the tunnel to the coordinator pod. The is only alive while the kubectl
+   process is running, so you can just kill the ``kubectl`` process that's
+   forwarding the port. In most cases that'll just mean pressing ``CTRL`` + ``C``
+   in the terminal where the port-forward command is running.
+
+Configuration
+-------------
+
+The helm chart uses the :doc:`Trino container image </installation/containers>`
+by default. The Docker image already contains a default configuration to get
+started, and some catalogs to allow you to explore Trino. Kubernetes allows you
+to mimic a :doc:`traditional deployment </installation/deployment>` by updating
+the ``values.yaml`` file. It's important to understand how files such as the
+Trino configuration, JVM, and various :doc:`catalog properties </connector>` are
+configured in Trino before updating the values.
+
+.. _creating-your-own-values-yaml:
+
+Creating your own values.yaml
+"""""""""""""""""""""""""""""
+
+When you use your own ``values.yaml``, you only override the values you specify.
+The remaining properties use their default values. Add a ``values.yaml`` with
+the following configuration:
+
+.. code-block:: yaml
+
+   image:
+     tag: "|trino_version|"
+   server:
+     workers: 2
```

Comment:
what is the default? I thought default is already 2....

Reply:
but fair I'll change it to 3 :)